src/Microsoft.DotNet.ProjectModel/CompilationOutputFiles.cs

```diff
@@ -58,14 +59,27 @@ public string PdbPath
 
         public virtual IEnumerable<string> Resources()
         {
-            var resourceNames = Project.Files.ResourceFiles
-                .Select(f => ResourceUtility.GetResourceCultureName(f.Key))
-                .Where(f => !string.IsNullOrEmpty(f))
-                .Distinct();
+            IEnumerable<string> resourceCultureNames = null;
+            var compilationOptions = Project.GetCompilerOptions(Framework, Configuration);
+            if (compilationOptions.EmbedInclude == null)
```

Comment:
@ajaybhargavb i assume you're still cleaning up this piece now so I'll save the comments.